docs/UserGuide.adoc

```diff
@@ -542,52 +542,61 @@ then annotations are shown as well.
 
 === Adding reminder: `reminder`
 
-Adds a reminder to visit a bookmark. You can set the time for the reminder. The app
-will inform you at that time. You can also choose whether to open the bookmark.
+Adds a reminder to visit a bookmark. You can set the time for the reminder.
+You can also choose whether to open the bookmark.
 The default setting is open.
 
-Format: `reminder INDEX t/TIME_DATE [n/NOTE=Open]`
+Format: `reminder INDEX t/TIME_DATE [d/DESCRIPTION] [n/NOTE=Open]`
 
 * INDEX is the index of the bookmark that you want to add with a reminder.
  * TIME_DATE follows HHMM dd/mm/yyyy format.
+* DESCRIPTION is the description of the reminder.
 * NOTE is the setting of whether you want to open the bookmark.
```

Comment:
If I remember correctly, we intended `NOTE` to be the description that is displayed in the reminder list. The text "Open" is the default text if there is no description. For example: "*Open bookmark 1 by 23/9/19 18:00*". (Refer to the dashboard mockup [here](https://github.com/AY1920S1-CS2103T-T13-4/main/blob/master/docs/images/ui-screenshots/uimockup1_dashboard.png).) 

Since `DESCRIPTION` now handles this role, **can I clarify what the function of `NOTE` is**? :) Does it automatically open a bookmarked webpage at the specific time? If yes, I'm concerned that it might be a bit intrusive for the user... Perhaps we could have a pop-up reminder at the specified time instead.

In either case, I think renaming the parameter `n/NOTE` would make it easier for the user to understand. For example, if the purpose of `n/NOTE` is modified, it could be renamed to `p/POP_UP`. (With the default value of `p/POP_UP` being `true`.)

Reply:
Okay, then I just use n/NOTE